### PR TITLE
Bug 1830497: Use privileged SCC to replace runlevel

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

File: bindata/manifests/plugins/002-rbac.yaml [hunk 1]
```diff
@@ -10,3 +10,61 @@ kind: ServiceAccount
 metadata:
   name: sriov-device-plugin
   namespace: {{.Namespace}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (fedepaol) [inline bindata/manifests/plugins/002-rbac.yaml:14]: Could the role be the same for both the cni and the dp and then bind both accounts to the same role?

Comment (pliurh) [reply bindata/manifests/plugins/002-rbac.yaml:14]: Technically, yes. 
@zshi-redhat do you think DP and CNI pod will need different roles in the future?

Comment (zshi-redhat) [reply bindata/manifests/plugins/002-rbac.yaml:14]: @pliurh for the foreseeable future, I didn't see there is such need.

**Comment by pliurh** [comment]
@fedepaol @zshi-redhat Now one role is defined for both CNI and DP SA. PTAL.

**Pull request merged**